temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
```

Comment:
Avoid using `pdb` for debugging in committed code. It is better to remove it or replace it with proper logging.
```
Remove the line or replace it with proper logging.
```